hq/app/logic/SnykDisplay.scala

```diff
@@ -0,0 +1,50 @@
+package logic
+
+import play.api.libs.json._
+import utils.attempt.{Attempt, FailedAttempt, Failure}
+
+import scala.concurrent.ExecutionContext
+
+import model._
+import model.Serializers._
+
+object SnykDisplay {
+
+  def getOrganisation(s: String, requiredOrganisation: SnykOrganisationName): Attempt[SnykOrganisation] = {
+    val organisation = for {
+      orglist <- (Json.parse(s) \ "orgs").asOpt[List[SnykOrganisation]]
```

Comment:
These all still throw runtime exceptions is the JSON is not well-formed.

Reply:
Done.